Formula/m/monocle.rb

```diff
@@ -0,0 +1,17 @@
+class Monocle < Formula
+  desc "See through all BGP data with a monocle"
+  homepage "https://github.com/bgpkit/monocle"
+  url "https://github.com/bgpkit/monocle/archive/refs/tags/v0.7.0.tar.gz"
+  sha256 "2ef5030b906a567b6fba2bc4aa593b8cfaa7e64c28557adc2ced60c89ff2a95d"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: ".")
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args
```

Reply:
Thanks, I updated the script following your suggestion.

Reply:
maybe good to also include lockfile for the cargo build, https://github.com/bgpkit/monocle/pull/66